pkg/apis/addtoscheme_integreatly_v1alpha1.go

```diff
@@ -5,6 +5,8 @@ import (
 	aerogearv1 "github.com/integr8ly/integreatly-operator/pkg/apis/aerogear/v1alpha1"
 	"github.com/integr8ly/integreatly-operator/pkg/apis/integreatly/v1alpha1"
 	kafkav1 "github.com/integr8ly/integreatly-operator/pkg/apis/kafka.strimzi.io/v1alpha1"
+
+	nexusv1 "github.com/integr8ly/integreatly-operator/pkg/apis/gpte/v1alpha1"
```

Comment:
what does gpte stand for?

Reply:
[Red Hat _Global Partner & Technical Enablement_ DevOps and Automation ](https://github.com/redhat-gpte-devopsautomation)

It's the first part of the group name (`gpte.opentlc.com`), so that's what was used when scaffolding the api with `operator-sdk add api`